woocommerce/woocommerce-ios
# Orders badge mark II: Storage

Fixes #1140 

Second step in the implementation of the Orders Badge: Storage

*Note* This PR is temporarily against the branch `issue/1089-orders-badge-step1`. My bad. I will update the base branch when #1138 is merged

## Changes
- Added data model number 18
- Added coredata entities (PR #1138 implements the Networking model objects that we would persist in this very PR)
- Added convenience method to load the OrderCount by siteID

## Testing
- 👀  at the code
- Checkout and run the branch.  👀 at the logs, make sure there are no coredata migration errors or warnings
- Check that there are no new warnings introduced.
